lib/process-services-cloud/src/lib/task/start-task/components/people-cloud/people-cloud.component.ts

```diff
@@ -190,7 +190,7 @@ export class PeopleCloudComponent implements OnInit, OnChanges {
                 this.logService.error(error);
             }
             const isUserValid: Boolean = this.userExists(result);
-            return isUserValid ? new IdentityUserModel(result[0]) : user;
+            return isUserValid ? new IdentityUserModel(result[0] || result) : user;
```

Comment:
how is it possible that can e an array on a user?

Reply:
We have an array of Users when we find a user by email or by username and the api return more than one user

Reply:
I updated the way of finding the users